includes/admin/views/experiments/experiments.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Admin Experiments View
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.3
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+// Get current screen and experiments.
+$screen      = get_current_screen();
+$experiments = acf()->admin_experiments->get_experiments();
+?>
+<div class="wrap" id="scf-admin-experiments">
+	<h1 class="wp-heading-inline"><?php esc_html_e( 'Experiments', 'secure-custom-fields' ); ?></h1>
+	<hr class="wp-header-end">
+
+	<div class="scf-experiments-list">
+		<div class="scf-experiments-header">
+			<p><?php esc_html_e( 'Enable or disable beta features. These features are in development and may change in future releases.', 'secure-custom-fields' ); ?></p>
+		</div>
+		
+		<?php if ( empty( $experiments ) ) : ?>
+			<div class="scf-no-experiments">
+				<p><?php esc_html_e( 'No beta features are currently available.', 'secure-custom-fields' ); ?></p>
+			</div>
+		<?php else : ?>
+		<form method="post" action="">
```

Comment:
Action is missing here.

Reply:
We can be more explicit, leaving it empty was using the current admin url page.